grails-app/services/com/unifina/service/ClusterService.groovy

```diff
@@ -0,0 +1,57 @@
+package com.unifina.service
+
+import com.streamr.api.client.CanvasesPerNode
+import com.streamr.api.client.StreamrClient
+import com.unifina.domain.signalpath.Canvas
+import groovy.transform.CompileStatic
+
+class ClusterService {
+	CanvasService canvasService
+	StreamrClient client
```

Comment:
Same here, bean won't be injected unless name matches the bean name `streamrClient`